package.json

```
@@ -51,6 +49,7 @@
     "googleapis": "^87.0.0",
     "israeli-bank-scrapers-core": "3.10.3",
     "keytar": "^7.4.0",
+    "lint-staged": "^13.3.0",
```

Version `13.3.0` is actually require node 14, as you can see [here](https://github.com/okonet/lint-staged/blob/v13.3.0/package.json#L17), and we need to take one version down.
```suggestion
    "lint-staged": "~13.2.3",
```